kroxylicious-filters/kroxylicious-multitenant/pom.xml

```diff
@@ -41,6 +41,11 @@
             <artifactId>kroxylicious-filter-test-support</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.kroxylicious</groupId>
+            <artifactId>kroxylicious-runtime</artifactId>
+            <scope>test</scope>
```

Comment:
Given how this plays with the enforcer is this actually a legitimate dependency? Should it actually be a dependency of `kroxylicious-filter-test-support` and thus only here transitively?

Reply:
by default [bannedDependencies](https://maven.apache.org/enforcer/enforcer-rules/bannedDependencies.html) seaches transitives, which is fair enough, you probably want to prevent an unexpected runtime dependency creeping in transitively.

Reply:
I think you're right that we should have modules for testing Filters against the runtime, so the work multitenant is doing should probably come through some intermediate test classes.